SocialCareCaseViewerApi/V1/UseCase/FormSubmissionsUseCase.cs

```diff
@@ -219,6 +220,11 @@ public CaseSubmissionResponse ExecuteUpdateSubmission(string submissionId, Updat
             UpdateSubmissionState(updatedSubmission, request, worker);
             UpdateResidents(updatedSubmission, request);
 
+            if (request.PinnedAt != null)
```

Comment:
so, we just need to pass `?pinnedAt` into the request url with any value, right?